rakelib/jobs.rake

```diff
@@ -14,4 +14,109 @@ namespace :jobs do
   task download_facility_access_data: :environment do
     Facilities::AccessDataDownload.perform_async
   end
+
+  LIVE_FORM_TYPES = %w(1990 1995 1990e 5490 1990n 5495).map { |t| "22-#{t.upcase}" }.freeze
+  SPOOL_DATE = '2017-10-26'.to_date
+
+  desc 'Dry run removing saved claims on 2017-10-26'
+  task check_malformed_claims: :environment do
+    malformed = malformed_ids
+
+    edu_claims = malformed[:education_benefits_claims]
+    saved_claims = malformed[:saved_claims]
+    edu_submissions = malformed[:education_benefits_submissions]
+    confirmation_numbers = malformed[:confirmation_numbers]
+
+    total = edu_claims.length + edu_submissions.length + saved_claims.length
+
+    puts "Task would remove #{total} total rows"
+    puts "\teducation_benefits_claims: #{edu_claims.length}"
+    puts "\tsaved_claims: #{saved_claims.length}"
+    puts "\teducation_benefits_submissions: #{edu_submissions.length}"
+
+    write_confirmation_numbers(confirmation_numbers)
+  end
+
+  desc 'Delete malformed saved claims on 2017-10-26'
+  task delete_malformed_claims: :environment do
+    malformed = malformed_ids
+
+    edu_claims = malformed[:education_benefits_claims]
+    saved_claims = malformed[:saved_claims]
+    edu_submissions = malformed[:education_benefits_submissions]
+    confirmation_numbers = malformed[:confirmation_numbers]
+
+    total = edu_claims.length + edu_submissions.length + saved_claims.length
+
+    puts "Removing #{total} rows"
+
+    write_confirmation_numbers(confirmation_numbers)
+    EducationBenefitsSubmission.delete(edu_submissions)
+    EducationBenefitsClaim.delete(edu_claims)
+    SavedClaim.delete(saved_claims)
+
+    puts 'Finished'
+  end
+
+  desc 'Rerun spool file for 2017-10-26'
```

Comment:
Definitely. I'll get this added.

Reply:
Actually, since this just writes out the spool files to disk rather than uploading them, I'll just add logging to this with info about how many applications were generated for each RPO.

Did we want to manually upload the generated spool files to the SFTP server after we've verified them? I could also write a task that actually performs the SFTP upload, making this method itself more of a dry run like you mentioned.